guide/messaging-dispatcher-service/tests/messaging-dispatcher_test.bal

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/jms;
+import ballerina/log;
+import ballerina/mb;
+import ballerina/runtime;
+import ballerina/task;
+import ballerina/test;
+
+documentation { Topic to publish Priority 2 messages. }
+endpoint jms:SimpleTopicPublisher SetoffAlarm {
+    initialContextFactory: "bmbInitialContextFactory",
+    providerUrl: "amqp://admin:admin@carbon/carbon"
+        + "?brokerlist='tcp://localhost:5672'",
```

Comment:
Merge with the line above.

Reply:
Merge with the line above it and format.